KrampitzKreutzwiserSettlersOfCatan/src/dataFiles/roadData.txt

```diff
@@ -0,0 +1,437 @@
+0
+0
+0
+0
+0
+
+0
```

Comment:
uhhhhm what is this?

Reply:
Thats the file of road node data
right now its full of zeros just to make sure the method doesnt crash and it can successfully load the data
That zero'd data creates all 72 nodes but they're all in the center of the screen and linked to the first settlement
My next step is to fill that with real data

Reply:
Ah got it, that's what you were talking about earlier, riiiight.